docs/sources/dashboards/share-dashboards-panels/shared-dashboards/index.md

```diff
@@ -0,0 +1,302 @@
+
+1. Click **Dashboards** in the main menu.
+1. Click the dashboard you want to share.
+1. Click the **Share** drop-down in the top-right corner and select **Share externally**.
+1. Do one of the following:
+   - Click **Pause access** so that people can't access the dashboard, but the link is maintained.
+   - Click **Resume access** so that people can access the dashboard again.
+   - Click **Revoke access** so that people can't access the dashboard unless a new external link is generated. Confirm that you want to revoke the link.
+1. Click the **X** at the top-right corner to close the share drawer.
+
+The link no longer works. You must create a new external URL, as in [Share externally to anyone with a link](#share-externally-to-anyone-with-a-link).
+
+## Share externally with specific people
+
+{{< admonition type="note" >}}
+Available in [Grafana Enterprise](ref:grafana-enterprise) and [Grafana Cloud](/docs/grafana-cloud).
+{{< /admonition >}}
+
+To share with specific external users, you can send them a link by email. Use this option when you only want to share your dashboard with specific people instead of anyone who navigates to the link. When you use email sharing, recipients receive a one-time use link that's valid for **one hour**. Once the link is used, the viewer has access to the shared dashboard for **30 days**.
+
+When you share a dashboard with an email link, your organization is billed per user for the duration of the 30-day token, regardless of how many dashboards are shared. Billing stops after 30 days unless you renew the token.
+
+1. Click **Dashboards** in the main menu.
+1. Click the dashboard you want to share.
+1. Click the **Share** drop-down in the top-right corner and select **Share externally**.
+1. In the **Link access** drop-down, select **Anyone with the link**.
+1. Click the checkbox confirming that you understand payment is required to add users.
+1. Click **Accept**.
+1. In the **Invite** field, enter the email address of the person you want to invite and click **Invite**.
+
+   You can only invite one person at a time.
+
+1. (Optional) Set the following options:
+   - **Enable time range** - Allow people accessing the link to change the time range. This configuration screen shows the default time range of the dashboard.
+   - **Display annotations** - Allow people accessing the link to view the dashboard annotations.
+1. Click **Copy external link**.
+1. Send the copied URL to any external user.
+1. Click the **X** at the top-right corner to close the share drawer.
+
+### Viewers requesting access
+
+If a viewer without access tries to navigate to the shared dashboard, they'll be asked to request access by providing their email. They'll receive an email with a new one-time use link if the email they provided has already been invited to view the shared dashboard and hasn't been revoked.
+
+If the viewer doesn't have an invitation or it's been revoked, you won't be notified and no link is sent.
```

Comment:
it's true no link is sent.
But you will never be notified, no matter the viewer has an invitation or not. 
I mean, it's a silence process.
If the user **is** in the email list, when he enters his email, he will receive a link **but** the "admin" won't be notified